docs/source/modin/supported/series_str_supported.rst

```diff
@@ -126,7 +126,7 @@ the method in the left column.
 | ``startswith``              | P                               |  ``N`` if the `na` parameter is set to a non-bool  |
 |                             |                                 |  value.                                            |
 +-----------------------------+---------------------------------+----------------------------------------------------+
-| ``strip``                   | Y                               |                                                    |
+| ``strip``                   | P                               | ``N`` if `to_strip` is non-string.                 |
```

Comment:
Just to confirm, this isn't a regression but rather was always the case and just not documented correctly?

Reply:
That's correct. Just improving the documentation.